### PR TITLE
feat: organization pytests 943

### LINKED ISSUE #943: LCFS - Update PyTests for Organization and Align Test Filenames
**Purpose:**  
To ensure the PyTests for the Organization feature are up to date with the latest changes in the Organization files and that each test file matches the corresponding file it is testing in the `organization/` directory.

**Acceptance Criteria:**
- [ ] Review the current Organization files to identify changes in methods and logic.
- [ ] Update the PyTest files to reflect the correct methods and signatures from the latest code.
- [ ] Ensure mock data and assertions align with the current implementation.
- [ ] Verify all Organization PyTests pass after updates.
- [ ] Rename test files to match the corresponding files in the `organization/` directory.

**Additional Context:**
- The previous PyTest files might include references to outdated methods, which should be updated to reflect the latest logic.
- Test logic should be rewritten to account for any changes in method arguments or return values.
- Ensure all test filenames align one-to-one with the corresponding files for better organization.

### PR DESCRIPTION
**Pull request by kevin-hashimoto**
- feat: pytests for fuel experts view, service, validation, closes #943 
- chore: commented out unused code. kept for context

aims for majority coverage

### CODE REVIEW

File: backend/lcfs/tests/organization/test_organization_views.py [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+    fastapi_app.dependency_overrides[TransferServices] = lambda: mock_transfer_services
+
+    response = await client.put(
+        url,
+        json=payload,
+    )
+
+    assert response.status_code == 201
+
+    # data = response.json()
```
Comment (dhaselhan) [inline backend/lcfs/tests/organization/test_organization_views.py:251]: Can we add these back?

File: backend/lcfs/tests/organization/conftest.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import pytest
```
Comment (dhaselhan) [inline backend/lcfs/tests/organization/conftest.py:1]: Love the use of conftest to keep the tests themselves clean!

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

File: backend/lcfs/tests/organization/test_organization_views.py (hunk 1 above)
Comment (kevin-hashimoto) [reply backend/lcfs/tests/organization/test_organization_views.py]: good catch! forgot about those

**Pull request merged**